continuous-integration/windows/build.py

```diff
@@ -26,7 +26,7 @@ def _run_process(args):
 
 
 def _build(root):
-    _run_process(("pip", "install", str(root)))
+    _run_process(("pip", "install", "--verbose", str(root)))
```

Comment:
This looks like a debug print, delete?

Reply:
Actually it was on purpose, because I wanted more output in the CI. If we had that, we would have caught the CI bug much sooner.

Do you think it should be removed?

Reply:
If it is useful, then keep it. Just add a line about it in the commit message

Reply:
Alright

Reply:
Done. Ready for next round.